portal_pages/forms.py

```diff
@@ -98,6 +100,37 @@ class Meta:
             'file'
         ]
 
+    def clean(self):
```

Comment:
Sorry I did not notice this when we were looking at this earlier, but on further thought it does not seem that this needs to be a form-level clean. We're just wanting to validate the file field of this form, so this should be done by a clean_file method not a form-level clean. See clean_<fieldname>() under https://docs.djangoproject.com/en/1.8/ref/forms/validation/#form-and-field-validation